src/acm_common/truststore_utils.py

```diff
@@ -125,6 +125,35 @@ def generate_keystore(
     logger.info("Java KeyStore generated")
 
 
+def add_cert_and_key(
+    priv_key, cert_list, alias, priv_key_password=None
+):
+    """Add certificate and private key.
+
+    Args:
+        priv_key (str): The base64 PEM-encoded private key to store
+        cert_list (List of str): A list of base64 PEM-encoded certificates
+            signed by ACM PCA.
+            Multiple certificates should represent the chain in the right order.
+        alias (str): The alias under which to store the key pair
+        priv_key_password (str): The password to protect the private key with
+
+    """
+    logger.info("Writing certificate and private key to filesystem")
+
+    f = open("/etc/pki/tls/private/" + alias + ".key", "a")
+    f.write(priv_key)
+    f.close()
+
+    for cert in cert_list:
+        f = open("/etc/pki/tls/certs/" + alias + ".crt", "a")
```

Comment:
This is slightly more pythonic written as:

```
with open("/etc/pki/tls/private/" + alias + ".key", "a") as f:
   f.write(priv_key)
```
(note there's no need for an explicit close; it'll be closed automatically when the `with` block completes)

Reply:
Updated filesystem writes to use with block